src/python/ensembl/io/genomio/events_dumper.py

```diff
@@ -25,10 +28,41 @@
 
 
 BRC4_START_DATE = datetime(2020, 5, 1)
+IdsSet = Set[str]
+DictToIdsSet = Dict[str, IdsSet]
+
+
+class Pair:
+    """Simple old_id - new_id pair representation"""
+
+    def __init__(self, old_id: Optional[str], new_id: Optional[str]) -> None:
+        """Create a pair with an old_id and a new_id if provided"""
+
+        if old_id is not None:
+            self.old_id = old_id
+        else:
+            self.old_id = ""
```

Comment:
Just for future reference, although I this alternative is more a preference than anything else:
```suggestion
        self.old_id = old_id if old_id is not None else ""
```